ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/Crosslink.java

```diff
@@ -47,6 +47,14 @@ public Crosslink(Crosslink crosslink) {
     this.data_root = crosslink.getData_root();
   }
 
+  public Crosslink() {
+    this.shard = UnsignedLong.ZERO;
+    this.start_epoch = UnsignedLong.ZERO;
+    this.end_epoch = UnsignedLong.ZERO;
+    this.parent_root = Bytes32.ZERO;
+    this.data_root = Bytes32.ZERO;
+  }
```

Comment:
Does this introduce a situation where the application might think that it is on `shard = 0`, `start_epoch = 0`, `end_epoch = 0`, `parent_root = 0` and `data_root = 0`. Basically a situation where the default value will be mistaken for a legitimate value?

Reply:
It probably might, but the spec had a place where it was initializing a new Crosslink object without passing any arguments, so I did this. How can we do this more safely?